Sources/PostgresNIO/New/PreparedStatement.swift

```diff
@@ -0,0 +1,40 @@
+/// A prepared statement.
+///
+/// Structs conforming to this protocol will need to provide the SQL statement to
+/// send to the server and a way of creating bindings are decoding the result.
+///
+/// As an example, consider this struct:
+/// ```swift
+/// struct Example: PostgresPreparedStatement {
+///     static var sql = "SELECT pid, datname FROM pg_stat_activity WHERE state = $1"
```

Comment:
Making sure our example is Sendable.
```suggestion
///     static let sql = "SELECT pid, datname FROM pg_stat_activity WHERE state = $1"
```